tests/ui/collapsible_span_lint_calls.stderr

```diff
@@ -0,0 +1,17 @@
+error[E0308]: mismatched types
+  --> $DIR/collapsible_span_lint_calls.rs:44:13
+   |
+LL |             db.span_help(expr.span, help_msg)
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^- help: try adding a semicolon: `;`
+   |             |
+   |             expected `()`, found `&mut rustc_errors::DiagnosticBuilder<'_>`
+
+error[E0308]: mismatched types
+  --> $DIR/collapsible_span_lint_calls.rs:47:13
+   |
+LL |             db.span_note(expr.span, note_msg)
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `()`, found `&mut rustc_errors::DiagnosticBuilder<'_>`
```

Comment:
These two forms emit error.

Reply:
Oh yeah. In that case `{ db._; }` is enough. Sorry for the confusion.

Reply:
The only case we should test should be `|db| { db._; }` only.